index.html

```diff
@@ -12,20 +12,31 @@
 <body>
     <header>
         <h1>Emilie Lokietz</h1>
+
         <nav>
             <a href="#kontakt">Kontakt</a>
             <a href="#lebenslauf">Biographie</a>
             <a href="">Mein Hund</a>
             <!-- da ist die Idee auf eine neue Website überzuleiten mit einer Galerie von den Fotos von meinem Hund-->
         </nav>
+
     </header>
 
-    <section>
-        <img src="hund.jpg" alt="Foto Hund" width="800" height="500">
-        <!--Das Photo ist von meinem Hund da ich aktuell keines von mir auf dem Mac habe-->
+    <section class="bild abstand">
```

Comment:
I'd recommend using English when declaring classes as that is the main language used by coders 🙂